package.json

```diff
@@ -56,6 +56,7 @@
     "moment": "^2.22.2",
     "moment-timezone": "^0.5.21",
     "react": "^15.6.2",
+    "jshint": "^2.9.6",
```

Comment:
Why did you add jshint as a dependencies. It should be a devDependencies but we already have eslint to do this task